Set_Demo/Card.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Card.swift
+//  Set_Demo
+//
+//  Created by Or Peleg on 24/04/2022.
+//
+
+import UIKit
+
+enum Shape: Int, CaseIterable { case triangle = 0, circle, square }
+enum Quantity: Int, CaseIterable { case one = 0, two, three }
+enum Color: Int, CaseIterable { case purple = 0, red, orange }
+enum Shading: Int, CaseIterable { case striped = 0, solid, open }
+
+final class Card: Hashable {
+    init(shape: Shape, quantity: Quantity, color: Color, shading: Shading) {
+        self.shape = shape
+        self.quantity = quantity
+        self.color = color
+        self.shading = shading
+        self.identifier = shape.rawValue * 1 + quantity.rawValue * 3 + color.rawValue * 9 + shading.rawValue * 27
+    }
+    func hash(into hasher: inout Hasher) {
+        hasher.combine(identifier)
+    }
+    static func == (lhs: Card, rhs: Card) -> Bool {
+            lhs.identifier == rhs.identifier
+    }
+    func unicodeValue() -> NSAttributedString {
```

Comment:
@or-peleg-healthy find a better place to put this functionality

Reply:
on my last commit which hasn't been pushed, func unicodeValue was added as an extension to Card on my viewController module